frontend/views/containers/chatroom/MessageNotification.vue

```diff
@@ -61,18 +63,28 @@ export default ({
           [MESSAGE_NOTIFICATIONS.UPDATE_DESCRIPTION]:
             L('Updated the channel description to: {description}', { description: channelDescription }),
           [MESSAGE_NOTIFICATIONS.DELETE_CHANNEL]: L('Deleted the channel: {title}', { title: channelName }),
-          [MESSAGE_NOTIFICATIONS.VOTE]: L('Voted on “{title}”', { title: '' }) // TODO: polls are not implemented yet
+          [MESSAGE_NOTIFICATIONS.VOTE_ON_POLL]: L('Voted on {options}', { options: votedOptions }),
+          [MESSAGE_NOTIFICATIONS.CHANGE_VOTE_ON_POLL]: L('Changed votes to {options}', { options: votedOptions })
```

Comment:
![Screen Shot 2023-07-25 at 11 14 17 AM](https://github.com/okTurtles/group-income/assets/138706/3904efcc-7638-4144-be99-6ec9a2397985)

Probably should read "Changed vote" (singular)

Reply:
Context:

![Screen Shot 2023-07-25 at 11 15 00 AM](https://github.com/okTurtles/group-income/assets/138706/d2424819-e234-40cf-a460-fd63d67b0f83)